public/app/features/explore/TraceView/components/TraceTimelineViewer/SpanDetail/index.tsx

```diff
@@ -236,6 +236,9 @@ export default function SpanDetail(props: SpanDetailProps) {
   }
 
   const focusSpanLink = createFocusSpanLink(traceID, spanID);
+  if (focusSpanLink) {
+    delete focusSpanLink.variableMap;
```

Comment:
`focusSpanLink` is destructured directly into the dom, so we remove a variable it is not expecting.